support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -81,6 +81,8 @@ DomainCreator:
         DecayTimescaleOuterBoundaryVelocity: 50.0
       RotationMap:
         InitialAngularVelocity: [0.0, 0.0, {{ InitialAngularVelocity }}]
+      TranslationMap:
+        InitialValues: [[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]]
```

Comment:
[optional] You could just make this `None` since we aren't using translation control at the moment. Same comment for the other input files

Reply:
I think I'll keep it in since the map functions change when you choose to not use a certain map, this way it works exactly like spec at the moment.